apps/site/components/EOL/EOLReleaseTable/TableClient.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import Switch from '@node-core/ui-components/Common/Switch';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+import { Fragment, useState } from 'react';
+
+import FormattedTime from '#site/components/Common/FormattedTime';
+import LinkWithArrow from '#site/components/Common/LinkWithArrow';
+import EOLModal from '#site/components/EOL/EOLModal';
+import VulnerabilityChips from '#site/components/EOL/VulnerabilityChips';
+import type { NodeRelease } from '#site/types/releases.js';
+import type { GroupedVulnerabilities } from '#site/types/vulnerabilities.js';
+
+type EOLReleaseTableBodyProps = {
+  eolReleases: Array<NodeRelease>;
+  vulnerabilities: GroupedVulnerabilities;
+};
+
+const EOLReleaseTableClient: FC<EOLReleaseTableBodyProps> = ({
+  eolReleases,
+  vulnerabilities,
+}) => {
+  const t = useTranslations();
+
+  const [currentModal, setCurrentModal] = useState<string | undefined>();
+  const [hideNonLts, setHideNonLts] = useState(false);
+
+  const filteredReleases = hideNonLts
+    ? eolReleases.filter(release => release.isLts)
+    : eolReleases;
+
+  return (
+    <>
+      <Switch
+        label={t('components.eolTable.hideNonLts')}
+        checked={hideNonLts}
+        onCheckedChange={setHideNonLts}
+      />
+      <table id="tbVulnerabilities">
```

Comment:
Why did you move this here and rename the file? Could you please revert this change? :)